Hakizimana-Clement/Advizr
# [Feature]: Implement Responsive and Reusable Card UI Component

#### PR Description
This PR introduces a responsive and reusable Card UI component designed to work seamlessly across different device sizes. The component structure has been optimized for reusability, allowing for easy integration in other parts of the application.

### What does this PR do?
New Feature about Card UI, and make resuable components

### Description of Task to be Completed
- [x] Task 1: Develop a responsive Card UI component that adjusts smoothly to various screen sizes.
- [x] Task 2: Refactor the component to use a reusable layout for consistency across the app.
- [x] Task 3: Integrate React Icons for easy use of icons within the Card component.

### How Should This Be Manually Tested?
1. Clone the repository and navigate into the project directory.
2. Check out the branch ft-card-ui and install dependencies with npm install.
3. Start the application locally by running npm run dev.
4. Navigate to the homepage and verify that the Card UI is responsive across different screen sizes.

### Any Background Context You Want to Provide?
The react-icons package was added to the project to simplify the process of adding and customizing icons within components.

### What Are the Relevant Pivotal Tracker Stories?
Feature #4: Card UI Component
